CHANGELOG.md

```diff
@@ -17,6 +17,8 @@
 
 - Updated deserialization of `bitmap` for custom CPU templates to allow usage
   of '_' as a separator.
+- Set FDP_EXCPTN_ONLY bit (CPUID.7h.0:EBX[6]) and ZERO_FCS_FDS bit
+  (CPUID.7h.0:EBX[13]) in Intel's CPUID normalization process.
```

Comment:
Just to doublecheck, are we sure this is Intel-specific?

Reply:
These bits are marked as reserved in AMD APM and added as Intel's de-feature bits in this commit in upstream kernel: https://github.com/torvalds/linux/commit/cbb99c0f588737ec98c333558922ce47e9a95827